update-stable-ids/Jenkinsfile

```diff
@@ -0,0 +1,105 @@
+import groovy.json.JsonSlurper
+
+pipeline {
+	agent any
+	
+	stages {
+		stage('Check upstream builds succeeded'){
```

Comment:
Similar to other sections - "Check upstream ConfimReleaseConfig build succeeded"?